script/run_tester

```diff
@@ -7,10 +7,7 @@
 BASE_DIRECTORY=`pwd`
 
 cd ${BASE_DIRECTORY}/test/feature
-docker kill tester > /dev/null 2>&1 || true
```

Comment:
I think you don't need to remove this line :)
Killing and removing container before running is one of docker practice to run successfully :point_up: 
This is referenced to CoreOS's systemd usage.

```
[Unit]
Description=MyApp
After=docker.service
Requires=docker.service

[Service]
TimeoutStartSec=0
ExecStartPre=-/usr/bin/docker kill busybox1
ExecStartPre=-/usr/bin/docker rm busybox1
ExecStartPre=/usr/bin/docker pull busybox
ExecStart=/usr/bin/docker run --name busybox1 busybox /bin/sh -c "while true; do echo Hello World; sleep 1; done"

[Install]
WantedBy=multi-user.target
```
https://coreos.com/docs/launching-containers/launching/getting-started-with-systemd/